models/CustomGallery.php

```diff
@@ -45,11 +45,7 @@ public function getPreviewImageUrl()
             return $path;
         }
         // get preview image from the file list
-        $media = $this->mediaListQuery()
-                ->orderBy([
-                    'sort_order' => SORT_ASC
-                ])
-                ->one();
+        $media = $this->mediaListQuery()->one();
```

Comment:
Why did you remove the order here? Shouldn't we show either the last or first image sorded by sort_order?